aiida/backends/djsite/db/migrations/0046_inline_calc_attributes.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Migrate attributes from old InlineCalculation that have been transformed into CalcFunctions:
+
+* OLD `sealed` and `function_name` remain the same
```

Comment:
the `sealed` attribute is not really particular to legacy `InlineCalculation` nodes so not sure we should even mention it.

Reply:
Mmm I'm not sure I understand why this is a factor. I feel that, in this case at least, there is more value in the completeness of being explicit about what I'm doing with all attributes than in saving two words in the docstring, no?

Reply:
Because to me it suggests that legacy `InlineCalculation` already used to have `sealed` nodes and it so happens that doesn't have to be changed. But this is not True, the `sealed` attribute has been added in some other migration.